### PR TITLE
Freezeimage

### PR DESCRIPTION
**Pull request by NimaSoroush**
- Adding evaluate functionality to evaluate expression on browser
- Adding freeze image functionality to freeze animations

### CODE REVIEW

File: src/freezeImage.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const freezeImage = (selector) => {
+  const anchor = document.querySelector(selector);
+  const src = anchor.src;
+  const className = anchor.className;
+  const canvas = document.createElement('canvas');
+  canvas.className = className;
+  const context = canvas.getContext('2d');
+  anchor.parentNode.replaceChild(canvas, anchor);
+
+  const image = new Image();
```
Comment (danrr) [inline src/freezeImage.js]: Rather than load the image again into a new Image, can we set onLoad on the image in the DOM and use that?

Comment (danrr) [inline src/freezeImage.js]: Also, we're trusting that the element is an image, might be worth checking

Comment (NimaSoroush) [inline src/freezeImage.js]: We are trying to redraw the new image and we replace the image tag. Apparently, this is how the drawImage works

Comment (NimaSoroush) [inline src/freezeImage.js]: What else could load image

**Pull request merged**